modules/tensor_mechanics/test/tests/interaction_integral/interaction_integral_3d_as_2d.i

```diff
@@ -0,0 +1,145 @@
+#This tests the J-Integral evaluation capability.
```

Comment:
J-integral -> interaction integral.  There are probably others like this as well.